sheetstorm/management/commands/data_load.py

```diff
@@ -0,0 +1,278 @@
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.utils import timezone
+
+from employees.factories import ReportFactory
+from employees.models import TaskActivityType
+from managers.factories import ProjectFactory
+from managers.models import Project
+from users.factories import UserFactory
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+    help = "Create initial sample data for SheetStorm application testing."
+
+    def add_arguments(self, parser):
+        #Users
+        parser.add_argument('-a','--admins', type=int, help='Indicates the number of admins to be created')
+        parser.add_argument('-e','--employees', type=int, help='Indicates the number of employees to be created')
+        parser.add_argument('-m','--managers', type=int, help='Indicates the number of managers to be created')
+        #Projects
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        #
+        # Users
+        #
+        self.stdout.write(f"Number of users in db before add {User.objects.count() }")
+        super_admin = User.objects.filter(email=f"admin@codepoets.it").first()
+        if super_admin is None:
+            super_admin = UserFactory(
+                email=f"admin@codepoets.it", user_type=User.UserType.ADMIN.name, is_superuser=True, is_staff=True,
+                first_name="Admin",
+                last_name="Kowalski"
+            )
+            super_admin.set_password("superuser")  # pylint: disable=no-value-for-parameter
+            super_admin.save()
+            self.stdout.write(f"Add super user")
```

Comment:
Ok, this is super pedantic of me, but I still wanted to point it out, wondering whether it shouldn't be written together as _superuser_. I am not quite sure myself. 😁  @kbeker ?

Reply:
done